managers/muse.py

```diff
@@ -93,7 +93,7 @@ def addReadableLinks(self):
         if self.pdfWebpubManifest:
             self.s3PDFReadPath = 'manifests/muse/{}.json'.format(self.museID)
             self.record.add_has_part_link(
-                self.createManifestInS3(self.s3PDFReadPath),
+                self.create_manifest_in_s3(self.s3PDFReadPath),
```

Comment:
Are there other ingest processes that have the exact same code to create the manifest?

Reply:
Yeah CLACSO, MET, DOAB, etc. have the same code for creating the manifest which is why I updated the name of the method in all of those processes in my commits.